stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/repository/domain/message/attachment/AttachmentToUploadEntity.kt

```diff
@@ -0,0 +1,33 @@
+package io.getstream.chat.android.offline.repository.domain.message.attachment
+
+import androidx.room.ColumnInfo
+import androidx.room.Embedded
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+
+@Entity(tableName = "attachment_to_upload")
```

Comment:
Shouldn't it have a foreign key `messageId`?

Reply:
This way it would compete with `AttachmentEntity` this table is only for the upload of attachment which we will later subscribe to.

Reply:
It won't compete. It's just foreign key for message table. BTW, let's see how we will work in the future with attachment uploading